src/content/language/keywords.md

```diff
@@ -10,123 +10,28 @@ nextpage:
-[factory]: /language/constructors#factory-constructors
-[false]: /language/built-in-types#booleans
-[final (variable)]: /language/variables#final-and-const
-[final (class)]: /language/class-modifiers#final
-[finally]: /language/error-handling#finally
-[for]: /language/loops#for-loops
-[Function]: /language/functions
-[get]: /language/methods#getters-and-setters
-[hide]: /language/libraries#importing-only-part-of-a-library
-[if]: /language/branches#if
-[implements]: /language/classes#implicit-interfaces
-[import]: /language/libraries#using-libraries
-[in]: /language/loops#for-loops
-[interface]: /language/class-modifiers#interface
-[is]: /language/operators#type-test-operators
-[late]: /language/variables#late-variables
-[library]: /language/libraries
-[mixin]: /language/mixins
-[new]: /language/classes#using-constructors
-[null]: /language/variables#default-value
-[on]: /language/error-handling#catch
-[operator]: /language/methods#operators
-[part]: /guides/libraries/create-packages#organizing-a-package
-[required]: /language/functions#named-parameters
-[rethrow]: /language/error-handling#catch
-[return]: /language/functions#return-values
-[sealed]: /language/class-modifiers#sealed
-[set]: /language/methods#getters-and-setters
-[show]: /language/libraries#importing-only-part-of-a-library
-[static]: /language/classes#class-variables-and-methods
-[super]: /language/extend
-[switch]: /language/branches#switch
-[sync]: /language/functions#generators
-[this]: /language/constructors
-[throw]: /language/error-handling#throw
-[true]: /language/built-in-types#booleans
-[try]: /language/error-handling#catch
-[type]: /language/extension-types
-[typedef]: /language/typedefs
-[var]: /language/variables
-[void]: /language/built-in-types
-[when]: /language/branches#when
-[with]: /language/mixins
-[while]: /language/loops#while-and-do-while
-[yield]: /language/functions#generators
-
-Avoid using these words as identifiers.
-However, if necessary, the keywords marked with superscripts can be identifiers:
-
-* Words with the superscript **1** are **contextual keywords**,
-  which have meaning only in specific places.
-  They're valid identifiers everywhere.
-
-* Words with the superscript **2** are **built-in identifiers**.
-  These keywords are valid identifiers in most places,
-  but they can't be used as class or type names, or as import prefixes.
-
-* Words with the superscript **3** are limited reserved words related to
-  [asynchrony support][].
-  You can't use `await` or `yield` as an identifier
-  in any function body marked with `async`, `async*`, or `sync*`.
-
-All other words in the table are **reserved words**,
-which can't be identifiers.
-
-[asynchrony support]: /language/async
+{% assign ckw = '&nbsp;<sup>1</sup>' %}
+{% assign bii = '&nbsp;<sup>2</sup>' %}
+
+The following table lists the words
+that the Dart language reserves for its own use.
+Don't use these terms as identifiers unless the term notes an exception.
```

Comment:
I think some form of the original guidance is appropriate here.  It's almost *never* good style to use any of these as identifiers.  So maybe say something like "These words cannot be used as identifiers unless otherwise noted.  Even when allowed, using keywords as identifiers is confusing and should be avoided."?

Reply:
I don't see a reason to discourage the user of, fx, `show` and `hide` as variable or method names. Sometimes they are precisely what you want to use, and they work.

The thing users _need_ to know are the actual restrictions: reserved words that cannot be used as a name, ever, and built-in identifiers that cannot be used as a type or prefix name.

The rest are not really important.
Or that many: `hide`, `show`, `async`, `sync` and `on`. Now possibly also `type`.
These are identifiers that can be used as names, even for types or prefixes (but as lower case, they shouldn't be names of types or extensions), but which also occur in a few specific places _not_ as a name to be resolved, but as a keyword of a syntactic construct.

If this is a list of every word that occurs as a "keyword" (managing: not as a resolvable name) anywhere in the grammar, then those words should be included, and linked to the descriptions of the construct**s** where the word can occur.
That's a fine index, if someone sees a word they don't understand, they can look it up, and see what it means in the context where they found it.

That index can also double as the list of restrictions, saying which words are restricted words and built-in identifiers, wish it's useful information. It can even mention that `type` cannot be the name of an `extension`, as a unique restriction.
I don't think being on the list as a contextual keyword should imply that the word should not be used as a name.
(And `await` and `yield` should be listed as restricted words, with a comment that they're not reserved everywhere, because users _should_ refrain from using them as names everywhere. Unlike `show` or `sync`, which are perfectly fine names.)